docs/sdk-and-tools/indices/transactions.md

```diff
@@ -0,0 +1,81 @@
+---
+id: transactions
+title: transactions
+---
+
+The `_id` field for this index is composed of transaction hash hex encoded.
+(example: `cad4692a092226d68fde24840586bdf36b30e02dc4bf2a73516730867545d53c`)
+
+| Field             | Description                                                                                                                            |
+|-------------------|----------------------------------------------------------------------------------------------------------------------------------------|
+| miniBlockHash     | The miniBlockHash represents the hash of the miniblocks in which the transaction is included.                                          |
+| nonce             | The nonce field represents sequence number of transactions sent from a given address.                                                  |
+| round             | The round field represents the round of block when transaction is executed.                                                            |
+| value             | The value field represents the amount of EGLD from the sender to the receiver.                                                         |
```

Comment:
The value field represents the number of EGLD to be sent from the sender to the receiver

Reply:
done